### PR TITLE
KK-267: Add Pushgateway service to feed metrics from jobs to Prometheus

### PR DESCRIPTION
**Pull request by HugoAreias**

### CODE REVIEW

**Comment by lukewm** [comment]
@SergioAreias if you're happy with this, please merge.

File: manifests/prometheus-pushgateway/prometheus-pushgateway.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      k8s-app: prometheus-pushgateway
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        k8s-app: prometheus-pushgateway
+        metrics-group: prometheus-pushgateway
+    spec:
+      containers:
+      - image: prom/pushgateway:PROMETHEUS_PUSHGATEWAY_VERSION
```
Comment (SergioAreias) [inline manifests/prometheus-pushgateway/prometheus-pushgateway.yaml]: You need a sed command exactly like all the others in order for this to work.

e.g. `sed -i -e 's/GRAFANA_VERSION/'"$GRAFANA_VERSION"'/g' manifests/grafana/grafana.de.yaml`

Comment (HugoAreias) [reply manifests/prometheus-pushgateway/prometheus-pushgateway.yaml]: added

File: deploy [hunk 2]
```diff
@@ -266,6 +271,9 @@ sed -i -e 's,PROMETHEUS_EXTERNAL_URL,'"$PROMETHEUS_EXTERNAL_URL"',g' manifests/p
 #set alertmanager external url
 sed -i -e 's,ALERTMANAGER_EXTERNAL_URL,'"$ALERTMANAGER_EXTERNAL_URL"',g' manifests/alertmanager/alertmanager.yaml
 
+#set prometheus pushgateway version
+sed -i -e 's/PROMETHEUS_PUSHGATEWAY_VERSION/'"PROMETHEUS_PUSHGATEWAY_VERSION"'/g' manifests/prometheus-pushgateway/prometheus-pushgateway.yaml
```
Comment (SergioAreias) [inline deploy]: Missing the $ sign for the variable replacement

Comment (HugoAreias) [reply deploy]: should have gone for search and replace...

**Pull request merged**